dist/style.css

```diff
@@ -59,3 +55,288 @@ body {
   -ms-transform: translate(-50%, -50%);
   transform: translate(-50%, -50%);
 }
+
+/* container css for most high level containers  */
+.container-parent {
+  height: 100%;
+  position: relative;
+}
+
+.container-header {
+  position: relative;
+  display: flex;
+  flex-direction: row;
+}
+
+.container-navSection {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding-bottom: 4%;
+}
+
+.container-nav {
+  font-family: 'AlexandriaFLF';
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-top: 5%;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.container-moto {
+  font-family: 'AlexandriaFLF';
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  width: 100%;
+  justify-content: left;
+}
+
+/* effects for hover on buttons */
+.menu {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  background-color: white;
+  text-align: center;
+  padding: 2%;
+  border-radius: 25px;
+  border: 2px solid #A9A9A9;
+  width: 25%;
+}
+
+.menu:hover {
+  background-color: #662A61;
+  color: white;
+}
+
+a {
+  text-decoration: none;
+}
+
+.fas:hover {
+  background-color: #562554;
+  color: white;
+}
+
+p {
+  margin: 0px;
+  margin-left: 7.5%;
+  margin-right: 10.5%;
+}
+
+fas {
+  height: 0%;
+}
+
+/* logo resizing  */
+.container-logo {
+  max-width: 65%;
+  height: auto;
+}
+
+#mainlogo {
+  max-width: 65%;
+  height: auto;
+  padding-right: -25%;
+}
+
+
+/* main content in split boxes  */
+.container-body {
+  margin-bottom: 20px;
+}
+
+.content {
+  font-family: 'AlexandriaFLF';
+  display: flex;
+  flex-direction: row;
+  border: 5px solid #562554;
+  border-radius: 25px;
+  background-color: white;
+}
+
+.content-left {
+  display: flex;
+  flex-direction: column;
+  margin: 2%;
+  width: 50%;
+  align-items: center;
+}
+
+.content-right {
+  display: flex;
+  flex-direction: column;
+  margin: 2%;
+  width: 50%;
+  align-items: center;
+  text-align: center;
+}
+
```

Comment:
This would be a great place to add a selector for the new div I asked you to create to contain the two `<h2>` elements at the top of `.content-right`.

If you chose not to go with that strategy, that's okay. In that case, you might want to consider a selector like `.content-right > h2`, to select only the `<h2>` tags inside this right column.

However you choose to code the HTML of this section, we're going to need to update the styling of these headers to satisfy the following:

- Update the font to match the mockup
- Make the entire text uppercase
- Reduce the space surrounding each tag to match the mockup (These h2's sure have a lot of `margin`!)

Reply:
I changed them from being an H2 and imported a font from google fonts - it's not the same though. Is there a way to find exactly what this font is?

Reply:
The font is provided for you in the repository already. Check the `assets/fonts` directory, or the fonts section at the top of the CSS file. 😄

Reply:
Thanks! I should've commented on this when I figured it out. I realized that the text was in all caps, it didn't look right until I capitalized all letters.